Libs/MRML/Widgets/qMRMLSliceWidget.h

```diff
@@ -85,27 +85,27 @@ class QMRML_WIDGETS_EXPORT qMRMLSliceWidget : public qMRMLWidget
 
   /// \sa qMRMLSliceControllerWidget::sliceViewName()
   /// \sa setSliceViewName()
-  QString sliceViewName()const;
+  Q_INVOKABLE QString sliceViewName()const;
 
   /// \sa qMRMLSliceControllerWidget::sliceViewName()
   /// \sa sliceViewName()
-  void setSliceViewName(const QString& newSliceViewName);
+  Q_INVOKABLE void setSliceViewName(const QString& newSliceViewName);
```

Comment:
Why not make these properties instead?
I think we usually define properties for variable types whenever it is possible, and for QString it should be possible.